test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -224,6 +226,14 @@ class HystrixSinkTest : public testing::Test {
   NiceMock<Upstream::MockClusterManager> cluster_manager_;
 };
 
+class AdminStreamMock : public Server::AdminStream {
```

Comment:
nit: this class should probably go under https://github.com/envoyproxy/envoy/blob/master/test/mocks/server/mocks.h